modules/reference/pages/properties/broker-properties.adoc

```diff
@@ -425,6 +425,22 @@ Path to the API specifications for the HTTP Proxy API.
 
 ---
 
+=== mode_mutability
+
+Allow modifying mode.
```

Comment:
This description is not helpful. Could we add what is mode and why would users want to modify it ? 
Property found at `src/v/pandaproxy/schema_registry/configuration.cc`

cc @BenPope

Reply:
This is the feature: https://github.com/redpanda-data/redpanda/pull/17952

When true, the Schema Registry and subjects can be changed between READONLY and READWRITE.